sci-biology/spm/ChangeLog

```diff
@@ -0,0 +1,7 @@
+# ChangeLog for sci-biology/spm
+# Copyright 1999-2015 Gentoo Foundation; Distributed under the GPL v2
+# $Header: $
+
+  17 Feb 2015;  <chymera@gentoo.org> +metadata.xml, +spm-8.ebuild.orig,
```

Comment:
It is not about you getting credit, but for the users to know who break their system after installing a certain package. See
```
emerge -u --changelog --pretend world
```